### DISCUSSION TITLE
Virtual Boy Blue Retro recognized by Macintosh BT, not by Virtual Boy

### DISCUSSION THREAD

**Discussion by SinisterSticks** [darthcloud/BlueRetro#811] [Q&A]
When i plug a 9 volt 2 Amp plug that fits the barrel connector, the Macintosh's Bluetooth successfully  scans my Virtual Boy BT adapter, and I can access the programming section on blueretro.io, but the Virtual Boy doesn't power on and give the warning.

My Virtual Boy is a "Virtual Double Tapped" consolized machine with separate left and right eye VGA that's merged with a 90s era live video VGA editing board with the left eye in black and red at 50% transparency and a black and cyan right eye background layer.

I heard to fully diagnose it, you need to describe the sitch as well as possible. That's why I shared the fact I have it "Double Tapped"  

It does work as expected with a Nintendo-made AC Adapter.

The Bluetooth controller I'm using is a home-made fight stick plugged into the Xbox Adaptive Controller.

I don't know if you have to "set the controls" before the Bluetooth controller is recognized. I don't know if you have to sync properly with the Adaptive Controller before the VB BT works.  I don't know if The VB BT has the proper power cord provided or if I have to fend for yourself for power.

I tried to plug in the Official Virtual Boy power plug into the pigtail but it doesn't fit.

Also I don't know if "double tapping + bluetooth uses too much power/ is otherwise power-inconpatible.

I also noticed the Retro Onyx VB Forum is closed.  But they may have moved to a Virtual Boy Discord.  I'll try that too.

I just want to make sure the BlueRetro end is correct before I diagnose a VB-specific issue.

**Comment by darthcloud** [comment]
The doc describe how to interface with the VB
https://github.com/darthcloud/BlueRetro/wiki/BlueRetro-Cables-Build-Instructions#cable-schematic-14

You need to use external power supply.

And yes it work I retested last week with v1.9

**Comment by darthcloud** [comment]
If you are using RetroOnyx's BlueRetro get support from him directly.

I don't know the specifics of his HW.

**Comment by Koohiisan** [comment]
I have absolutely nothing to add to helping you find a solution, so sorry to hijack.  But I am fanatically interested in your setup.  I was told unequivocally that it was impossible to use two Virtual Taps to get 3D to a TV or monitor.  If you've found a way, even if it utilizes old tech for video output modifications, please for the sake of the VB document this somewhere online.  I know I'd love to see your setup and what you are using, and I have to believe other members of the VB community would love to as well.  That's my request, sorry again to hijack.

**Comment by SinisterSticks** [comment]
Hello I assume replying to this email will also simultaneously posted to the dark cloud blue retro forum.
  

  
They say always give the short answer first and give details later.
  

  
The short answer is '90s era Television Broadcast Studio video editing devices.
  

  
First of all I know why the original video by furtek did not show good 3D according to most people.
  

  
It was because the colors for Tech chose were red for the right eye left eye and blue for the right eye which when seen through red and cyan glasses (remember cyan is green light plus blue light or the primary pigment that blocks red light) the picture had a magenta-ish tint, which makes sense when you figure out the modern scientific color wheel.
  

  
The problem with that is that not all the light is oerfectly segregated.
  

  
So yes the makers of chromeadepth to look at the original furtek video with red and cyan glasses versus red and blue glasses and he doesn't admit that red and blue glasses properly show the 3D correctly.
  

  
I tried using VGA y adapters at first to combine the red of the left eye with the green and blue the right eye unfortunately they're vertical and horizontal sync issues which is done in that generic way would cause one screen to scroll like you have to adjust the horizontal hold.
  

  
Then I remembered me and my dad did TV shows for the local cable company in the 90s and dad had a consumer level video editor which combined videos so I thought maybe I could buy those videos into a red and cyan anaglyph.    However there was one nitpick and one glaringly obvious problem.    My editor took two composite video inputs and output in composite as the merged video.    That does two things, makes the picture look fuzzier with less pixels per screen and introduces ping somewhere.
  

  
I also noticed when I took a video adapter that had one VGA input and one composite input and one VGA output, the composite video was always a frame behind the VGA video,    therefore the 3d effect looked off.
  

  
So I tested to see if my composites video merger added any ping.    I figured pinball is one of the most reflex heavy games that doesn't rely on a pattern therefore I played NES pinball on the composite inputs and tried mixing it with another picture slightly in the background and lo and behold pinball played perfectly fine, (okay if you were playing the white gun game you might notice the ping from an aim shift, but other than that perfectly fine)
  

  
So I had to find a dual VGA in single VGA out adapter and I tried a couple different kinds and the one I found that was the cheapest available on eBay was an Analog Way Pulse PLS300.    80 bucks used guaranteed working on eBay plus 20 shipping.
  

  
I just had to learn how to program it using the button interface without a PC because I had a Macintosh and my Macintosh doesn't do Windows.
  

  
All you have to do is set one of the two eyes to be the background layer of this professional equipment set the other one to be the over layer, set the percentage of transparity of the over layer to 50% and what it live at 100% size and keep the resolution inputs and output the same because resolution translation adds ping.    And lo and behold I got my first set of 2 million plus games on Galactic pinball playing four games on four different tables in a row.
  

  
They're probably other devices from other companies that do similar things but the key thing you're looking for is two VGAs in one VGA out and the ability to work with certain VGA resolutions and stick within those resolutions, and finally the ability to turn one picture into a 50% transparent layer over the other.
  

  
For $100 this is a very broad purpose device.    He could do pictured picture we could do logo it could do other stuff like that.
  

  
If that machine was shrunk and simplified to simply be an antaglyph overlay then combined with two virtual Taps you have the Virtual double tap.
  

  
However, not all is great in the third dimension.
  

  
The Virtual Boy Bluetooth controller adapter from Blue retro apparently does not work on my Virtual Boy even though I got the proper cord,    the proper power source, but the Virtual Boy does work double tapped with the OEM Virtual Boy power adapter and control pad dock.
  

  
We're trying to figure out whether my individual Bluetooth Virtual Boy controller device is faulty or whether it's a fault specific to double tap virtual boys.
  

  
I don't have a single tapped Virtual Boy to test to see if it otherwise works.
  

  
So that means all I can play is the games with the pad, at least right now,    so therefore Street Fighter 2 would be tough.    It would have also been cool to play tele boxer with twin joysticks with thumb and finger triggers.
  

  
But for pinball the basic controller is pretty good because it's like you're hitting flippers on a pinball table.    Those shift buttons come in handy in that game.
  

  
Meanwhile we're trying to solve the mystery of why a Virtual Boy Bluetooth works with a single tap but not my particular double tapped Virtual Boy.    The diagnosis is my computer reads the Bluetooth portion it's the portion of deals with the virtual boy that's short on power that doesn't bring up the ocular warning screen.
  

  
So anyone who can't help with that let us know.    I tried a couple things like plugging the power directly in the wall socket and making sure there's room for it to be fully inserted and neither of those assurances fixed it.    So it probably is a power distribution issue.
  

  
I did not build the double tapped VB by myself.    I hired a guy to do it he was the recommended guy closest to me in New York.    He's a guy who professionally builds console mods for people.    Search New York Virtual Boy and console mods and you'll probably see his user name.    Ask him if he did anything special on my double tap virtual boy cuz he could have tested it to see if it worked and any modifications he done would go credit to him. I'm just the recipient of his handiwork.
  

  

  

  

  
  
  
  
>

**Comment by SinisterSticks** [comment]
It might me a specific power issue with a double tapped VB for 3d video.
  

  

  

  
  
  
  
  
>
